storage/txcache/txListForSenderAsSortedMapItem.go

```diff
@@ -0,0 +1,81 @@
+package txcache
+
+import (
+	"math"
+
+	"github.com/ElrondNetwork/elrond-go/storage/txcache/maps"
+)
+
+type senderScoreParams struct {
+	count uint64
+	// Size is in bytes
+	size uint64
+	// Fee is in micro ERD
+	fee uint64
+	gas uint64
+	// Price is in micro ERD
+	minGasPrice uint32
+}
+
+// GetKey return the key
+func (listForSender *txListForSender) GetKey() string {
+	return listForSender.sender
+}
+
+func (listForSender *txListForSender) getLastComputedScore() uint32 {
+	return listForSender.lastComputedScore.Get()
+}
+
+// ComputeScore computes the score of the sender, as an integer 0-100
+func (listForSender *txListForSender) ComputeScore() uint32 {
+	score := uint32(listForSender.computeRawScore())
+	listForSender.lastComputedScore.Set(score)
+	return score
+}
+
+func (listForSender *txListForSender) computeRawScore() float64 {
+	fee := listForSender.totalFee.GetUint64()
+	gas := listForSender.totalGas.GetUint64()
+	size := listForSender.totalBytes.GetUint64()
+	count := listForSender.countTx()
+	minGasPrice := listForSender.cacheConfig.MinGasPriceMicroErd
+
+	return computeSenderScore(senderScoreParams{count: count, size: size, fee: fee, gas: gas, minGasPrice: minGasPrice})
+}
+
+func computeSenderScore(params senderScoreParams) float64 {
+	allParamsDefined := params.fee > 0 && params.gas > 0 && params.size > 0 && params.count > 0
+	if !allParamsDefined {
+		return 0
+	}
+
+	PPUMin := float64(params.minGasPrice)
+	PPUAvg := float64(params.fee) / float64(params.gas)
+	PPUScore := math.Pow(PPUAvg/PPUMin, 3)
+
+	countPow2 := float64(params.count) * float64(params.count)
+	countScore := math.Log(countPow2+1) + 1
+
+	// We use size in ~kB
+	size := float64(params.size) / 1000
```

Comment:
1kb != 1000bytes (1000 - magic number)

Reply:
`kB`, for simplicity.
I've added `const bytesInKB = 1000` though.